src/main/kotlin/systems/ajax/malov/stockanalyzer/config/MongoConfig.kt

```diff
@@ -0,0 +1,9 @@
+package systems.ajax.malov.stockanalyzer.config
+
+import io.mongock.runner.springboot.EnableMongock
+import org.springframework.context.annotation.Configuration
+
+
+@Configuration
```

Comment:
```suggestion
import org.springframework.context.annotation.Configuration

@Configuration
```
detekt should complain about this